src/VisualStudio/Core/Def/Progression/GraphQueries/SearchGraphQuery.cs

```diff
@@ -39,7 +39,6 @@ public async Task<GraphBuilder> GetGraphAsync(Solution solution, IGraphContext c
             callback,
             searchPattern,
             NavigateToUtilities.GetKindsProvided(solution),
-            threadingContext.DisposalToken,
```

Comment:
Seems we could remove `threadingContext` from the type completely

Reply:
touche :)